src/install/cassandra/whatsNext.mdx

```diff
@@ -0,0 +1,1377 @@
+---
+componentType: default
+headingText: What's next?
+---
+
+<Callout variant="important">
+This integration doesn't automatically update. For best results, regularly [update the integration package](/docs/integrations/host-integrations/installation/update-infrastructure-host-integration-package) and [the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/installation/update-infrastructure-agent).
+</Callout>
+
+## Configuration options [#config-options]
+
+For the standard on-host installation, this integration comes with a YAML config file, `apache-config.yml`. This configuration is where you can place required login credentials and configure how data is collected. Which options you change depends on your setup and preferences. It comes with a sample config file `apache-config.yml.sample` that you can copy and edit.
```

Comment:
Looks like this is talking about the apache integration instead of the cassandra integration

Reply:
Yes, I know. I copied from here: [https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/cassandra-monitoring-integration/](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/cassandra-monitoring-integration/)

Reply:
Updated.